deployment/user_test.go

```diff
@@ -92,3 +92,80 @@ func TestAdd(t *testing.T) {
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), "A duplicate role binding already exists")
 }
+
+func TestUpdateUser(t *testing.T) {
+	testUtil.InitTestConfig()
+	okResponse := `{
+		"data": {
+			"deploymentUpdateUserRole": {
+				"id": "ckggzqj5f4157qtc9lescmehm",
+				"user": {
+					"username": "somebody@astronomer.com"
+				},
+				"role": "DEPLOYMENT_EDITOR",
+				"deployment": {
+					"releaseName": "prehistoric-gravity-9229"
+				}
+			}
+		}
+	}
+`
+	client := testUtil.NewTestClient(func(req *http.Request) *http.Response {
+		return &http.Response{
+			StatusCode: 200,
+			Body:       ioutil.NopCloser(bytes.NewBufferString(okResponse)),
+			Header:     make(http.Header),
+		}
+	})
+	api := houston.NewHoustonClient(client)
+	deploymentId := "ckggzqj5f4157qtc9lescmehm"
+	email := "somebody@astronomer.com"
+	role := houston.DeploymentEditor
+
+	buf := new(bytes.Buffer)
+	err := UpdateUser(deploymentId, email, role, api, buf)
+	assert.NoError(t, err)
+	assert.Contains(t, buf.String(), "Successfully updated somebody@astronomer.com to a DEPLOYMENT_EDITOR")
```

Comment:
I'm not sure what's happening in the tests, but I had to switch this to `contains` because only the tests results are showing records for the `Added` user from `line 91` and the `Updated` user from this test.

I think it might be a bug in the testSuite teardown/setup or something with the `buffer`, but I figured that using the `contains` here is OK.